contribs/gmf/src/controllers/AbstractDesktopController.js

```diff
@@ -242,7 +242,7 @@ export class AbstractDesktopController extends AbstractAPIController {
      * The ngeo ToolActivate manager service.
      * @type {import("ngeo/misc/ToolActivateMgr.js").ToolActivateMgr}
      */
-    const ngeoToolActivateMgr = $injector.get('ngeoToolActivateMgr');
+    this.ngeoToolActivateMgr = $injector.get('ngeoToolActivateMgr');
```

Comment:
Why this change?

Reply:
Otherwise, I've to redo `$injector.get('ngeoToolActivateMgr');` in the `desktop_alt/Controller.js` Nothing big, but I think that not cast anything to reference it as variable of `AbstractDesktopController.js` (to be able to do `this.ngeoToolActivateMgr....` in the `desktop_alt/Controller`)